src/index.js

```diff
@@ -79,11 +79,14 @@ class Dropzone extends React.Component {
       this.dragTargets.push(e.target);
     }
 
-    const allFilesAccepted = this.allFilesAccepted(getDataTransferItems(e, this.props.multiple));
+    // validate all files regardless if they are in multi file mode.
+    const files = getDataTransferItems(e, true);
```

Comment:
1. Shouldn't it be aligned with https://github.com/okonet/react-dropzone/pull/410/files#diff-1fdf421c05c1140f6d71444ea2b27638R133?
1. `true` is default argument, so you can skip it